cmd/sqlflowserver/main_test.go

```diff
@@ -621,3 +659,66 @@ INTO sqlflow_models.my_dnn_model;`
 	// call ParseRow only to wait train finish
 	ParseRow(stream)
 }
+
+// CaseTrainALPS is a case for training models using ALPS with out feature_map table
+func CaseTrainALPS(t *testing.T) {
+	a := assert.New(t)
+	trainSQL := fmt.Sprintf(`SELECT deep_id, user_space_stat, user_behavior_stat, space_stat, l
+FROM %s.sqlflow_sparse_feature_test
```

Comment:
The table in the `maxcompute` may not exist or without reading permission, why not creating a temporary table with fixed data before test running?

Reply:
Good shot, will do

Reply:
Done